node/src/components/deploy_buffer.rs

```diff
@@ -385,6 +385,18 @@ impl DeployBuffer {
             }
         }
         self.update_all_metrics();
+
+        info!(
+            "produced {}, buffer has {} held, {} dead, {} total",
+            ret,
+            self.hold
+                .values()
+                .map(|deploys| deploys.len())
+                .sum::<usize>(),
+            self.dead.len(),
+            self.buffer.len()
+        );
```

Comment:
As in Alex's PR, I'd prefer structured logging, but I won't block on that :wink: